client/src/store/reducer/employeeTableReducer.js

```diff
@@ -0,0 +1,169 @@
+const initalState = {
+  employeeData: undefined,
+  filteredEmployeeData: undefined,
+  filters: {
+    department: {},
+    office: {},
+    name: '',
+  },
+  recipients: [],
+  page: 0,
+  rowsPerPage: 10,
+};
+
+const toggleSingleRecipient = (array, item) => {
+  const itemIndex = array.indexOf(item);
+  if (itemIndex === -1) {
+    return [...array, item];
+  }
+
+  return array.filter((x) => x !== item);
+};
+const toggleRecipients = (array, checked, recipientsList) => {
+  let updatedArray = [...array];
+  if (checked) {
+    recipientsList.forEach((item) => {
+      const itemIndex = array.indexOf(item);
+      if (itemIndex === -1) {
+        updatedArray = [...updatedArray, item];
```

Comment:
since `updatedArray` is already a new version of `array`, you can just do updatedArray.push(item)